baiji/pod/asset_cache.py

```
@@ -102,8 +102,8 @@ def is_outdated(self):
     def download(self, verbose=True):
         try:
             s3.cp(self.remote, self.local, force=True, progress=verbose, validate=True)
-        except s3.KeyNotFound:
-            raise s3.KeyNotFound('{} not found on s3'.format(self.remote))
+        except s3.KeyNotFound as e:
+            raise s3.KeyNotFound('{0} not found on s3. Reason: {1}'.format(self.remote, str(e)))
```

Should we just re-raise the exception?